service/src/main/java/uk/gov/hmcts/reform/fpl/service/UploadC2DocumentsService.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class UploadC2DocumentsService {
```

Comment:
I am not biggest fan of this class and below, I think they are tightly coupled and there is not massive benefit of having them separate and wiring through the dependency injection (unless this one has more functionality as stated in the comment above in the controller. 

It lacks unit test as well.

Reply:
added unit test. Idea is for @Jamesnelson117 to use `ValidateSupportingEvidenceBundleService` for other supporting evidences he is working on ...

Reply:
Ah ok, cool that makes sense then.